pkg/pillar/cmd/diag/diag.go

```diff
@@ -142,7 +142,8 @@ func Run() {
 		DeviceNetworkStatus: ctx.DeviceNetworkStatus,
 		FailureFunc:         zedcloud.ZedCloudFailure,
 		SuccessFunc:         zedcloud.ZedCloudSuccess,
-		NetworkSendTimeout:  ctx.globalConfig.NetworkTestTimeout,
+		// I'm confused. Old code sets network send timeout to network test timeout. which side is correct?
```

Comment:
Current code setting it to ctx.globalConfig.NetworkTestTimeout is a bug.. It should be set to NetworkSendTimeout. you did the right thing..

Please remove the comment..